lib/services/networkManagement2/package.json

```diff
@@ -6,7 +6,7 @@
     "Rajendran, Deepak <derajen@microsoft.com>",
     "Choubey, Ankur <anchoube@microsoft.com>"
   ],
-  "version": "1.3.0-preview",
+  "version": "1.4.0-preview",
```

Comment:
yep, if a method that was shipped was renamed, then this is definitely a breaking change. 

How? Because the consumers of that method will be broken if they upgrade to the newer package. To keep their applications from being broken, they'd like their app to stick to a compatible version of the package. If we didn't follow semver's guidelines [rule 1 shown above applies here], we'd be breaking them.